src/style.css

```
@@ -0,0 +1,42 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  width: 96px;
+  height: 116px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: stretch;
+  margin-bottom: 4px;
+}
+
+.stars__star,
+.stars__star:first-child {
```

```suggestion
.stars__star {
```
Ты это применяешь ко всем звездам по умолчанию